resources.yml

```diff
@@ -10204,7 +10204,7 @@
   notes:
 
 - name: VMware Education
-  url: https://mylearn.vmware.com/mgrReg/plan.cfm?plan=33611&src=so_5a314d05ddb83&cid=70134000001SkJd
```

Comment:
I tried out these two URLs, the originals are correct... Not sure why these were to be changed...